apiary-metastore-events/kafka-metastore-events/pom.xml

```diff
@@ -75,18 +71,6 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
```

Comment:
I for one won't miss powermock so thanks for doing this!

Reply:
No worries, one of those things where if it doesn't happen now, it probably wont!